lib/options_test.go

```diff
@@ -367,6 +367,12 @@ func TestOptions(t *testing.T) {
 		opts := Options{}.Apply(Options{RunTags: tags})
 		assert.Equal(t, tags, opts.RunTags)
 	})
+	t.Run("DiscardResponseBody", func(t *testing.T) {
```

Comment:
That's good, but can you also add another test that checks the discard functionality itself? Something like [this](https://github.com/loadimpact/k6/blob/d4152d20221c0c466c93fe5cabb8e8f8c885cb5b/js/modules/k6/http/response_test.go#L88-L95), but checking that the response body is empty when you specify `discardResponseBody`?